Mage.Sets/src/mage/cards/l/Lightsaber.java

```diff
@@ -27,23 +32,30 @@
  */
 public final class Lightsaber extends CardImpl {
 
+    private static final FilterPermanent filter = new FilterControlledPermanent("Jedi or Sith");
+
+    static {
+        filter.add(Predicates.or(
+                SubType.JEDI.getPredicate(),
+                SubType.SITH.getPredicate()
+        ));
+    }
+
     public Lightsaber(UUID ownerId, CardSetInfo setInfo) {
         super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{1}");
         this.subtype.add(SubType.EQUIPMENT);
 
-        // Equiped creature gets +1/+0 and has firsttrike
+        // Equipped creature gets +1/+0 and has first strike.
         this.addAbility(new SimpleStaticAbility(new BoostEquippedEffect(1, 0)));
         this.addAbility(new SimpleStaticAbility(new GainAbilityAttachedEffect(
                 FirstStrikeAbility.getInstance(), AttachmentType.EQUIPMENT
         ).setText("and has first strike")));
 
-        // Equip 3
-        Ability ability = new EquipAbility(3);
-        ability.setCostAdjuster(LightsaberAdjuster.instance);
-        this.addAbility(ability);
+        // Equip {3}
+        this.addAbility(new EquipAbility(Outcome.AddAbility, new GenericManaCost(3)));
```

Comment:
Pull from master and merge in the changes to the equip ability. Then add a third parameter ,`false`, to the equip ability constructor to not display the hint text, which looks like it shouldn't display.